test/e2e/testdata/lots-of-configuration/tenant.yaml

```diff
@@ -738,3 +738,8 @@ logStreams:
         aws.partner/auth0.com/auth0-user-79f1a6de-7fc2-47de-9202-4b7b15c9d601/auth0.logs
     status: active
     type: eventbridge
+  - name: Suspended DD Log Stream
+    sink:
+      datadogApiKey: some-sensitive-api-key
+      datadogRegion: us
+    type: datadog
```

Comment:
The omission of `status` is key here. This simulates what a suspend log stream will look like when exported from Auth0 tenant to local configuration files.